src/BloomExe/Publish/EpubMaker.cs

```diff
@@ -137,11 +138,25 @@ public void StageEpub(bool publishWithoutAudio = false)
 					CopyStyleSheets(pageDom);
 			}
 
-			const string coverPageImageFile = "thumbnail-256.png";
+			string coverPageImageFile = "thumbnail-256.png";
 			// This thumbnail is otherwise only made when uploading, so it may be out of date.
 			// Just remake it every time.
 			_thumbNailer.MakeThumbnailOfCover(Book, 256, Form.ActiveForm);
-			CopyFileToEpub(Path.Combine(Book.FolderPath, coverPageImageFile));
+			var coverPageImagePath = Path.Combine(Book.FolderPath, coverPageImageFile);
+			if (!File.Exists(coverPageImagePath))
+			{
+				// Something went wrong...should have been reported already...use the low-res image we always have
```

Comment:
As a result of my internal "5 whys", I'd like us to get in the habit of simulating errors and making sure we like the user experience that occurs. I've found that it's often worse than I imagined, just looking the code. Have you done that for this error?

Conser if using the new nonfatalerror reporting would help. When I looked into this bug, it seemed like a mystery; I did not see the expected log for a time out, for example. In the error message, it would be helpful to have a reference to this iss number.